spec/spec_helper.rb

```diff
@@ -2,6 +2,7 @@
 require "xcake"
 
 require "coveralls"
+require 'pry'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.